docs/ClassDiagram.md

```diff
@@ -0,0 +1,13 @@
+# Class diagram
+- [x] In processing
+- [ ] Done
+
```

Comment:
Cái In processing này OK nè, có điều thêm vào trên cùng cái link về trang chủ ```[Trang chủ](/)```
Và cuối trang điều hướng đến các trang liên quan nếu có nha.